mattermost-plugin/webapp/src/components/FocalboardUnfurl.scss

```
@@ -0,0 +1,139 @@
+                color: #3D3C40;
+                font-weight: 600;
+                font-size: 14px;
+            }
+
+            .board_title {
+                color: rgba(61, 60, 64, 0.56);
+                font-size: 12px;
+                line-height: 16px;
+            }
+        }
+    }
+
+
+    .body {
+        border: 1px solid rgba(61, 60, 64, 0.16);
+        border-radius: 4px;
+        margin-top: 16px;
+        height: 145px;
+        overflow: hidden;
+        padding: 16px;
+        white-space: nowrap;
+
+        h1, h2, h3, h4, h5, h6 {
+            margin-top: 0;
+        }
+    }
+
+    .footer {
+        display: flex;
+        align-items: center;
+        height: 40px;
+        margin-top: 16px;
+
+        .timestamp_properties {
+            margin-left: 12px;
+            max-width: 90%;
+
+            .properties {
+                display: flex;
+                align-items: center;
+                white-space: nowrap;
+
+                .remainder {
+                    color: rgba(61, 60, 64, 0.48);
+                    font-weight: bold;
+                    font-size: 14px;
+                }
+
+                .property {
+                    border-radius: 4px;
+                    padding: 2px 4px;
+                    margin-right: 10px;
+                    max-width: 33%;
+                    overflow: hidden;
+                    text-overflow: ellipsis;
+                    overflow-wrap: normal;
+
+                    &.propColorDefault {
+                        background-color: var(--prop-default);
+                    }
+                    
+                    &.propColorGray {
+                        background-color: var(--prop-gray);
+                    }
+                    
+                    &.propColorBrown {
+                        background-color: var(--prop-brown);
+                    }
+                    
+                     &.propColorOrange {
+                        background-color: var(--prop-orange);
+                    }
+                    
+                    &.propColorYellow {
+                        background-color: var(--prop-yellow);
+                    }
+                    
+                    &.propColorGreen {
+                        background-color: var(--prop-green);
+                    }
+                    
+                    &.propColorBlue {
+                        background-color: var(--prop-blue);
+                    }
+                    
+                    &.propColorPurple {
+                        background-color: var(--prop-purple);
+                    }
+                    
+                    &.propColorPink {
+                        background-color: var(--prop-pink);
+                    }
+                    
+                     &.propColorRed {
+                        background-color: var(--prop-red);
+                    }
+                }
+               
+            }
+
+
+
+        }
+
+    }
+
+}
```

nit - remove new lines.
```suggestion
                }             
            }
        }
    }
}
```